sup3r/models/wind_conditional_moments.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+"""Wind conditional moment estimator with handling of low and
+high res topography inputs."""
+import numpy as np
+import logging
+import tensorflow as tf
+from phygnn.layers.custom_layers import Sup3rAdder, Sup3rConcat
+
+from sup3r.models.conditional_moments import Sup3rCondMom
+
+
+logger = logging.getLogger(__name__)
+
+
+class WindCondMom(Sup3rCondMom):
+    """Wind conditional moment estimator with handling of low and
+    high res topography inputs.
+
+    Modifications to standard Sup3rCondMom:
+        - Hi res topography is expected as the last feature channel in the true
+          data in the true batch observation.
+        - If a custom Sup3rAdder or Sup3rConcat layer (from phygnn) is present
+          in the network, the hi-res topography will be added or concatenated
+          to the data at that point in the network during either training or
+          the forward pass.
+    """
+
+    def set_model_params(self, **kwargs):
+        """Set parameters used for training the model
+
+        Parameters
+        ----------
+        kwargs : dict
+            Keyword arguments including 'training_features', 'output_features',
+            'smoothed_features', 's_enhance', 't_enhance', 'smoothing'
+        """
+        output_features = kwargs['output_features']
+        msg = ('Last output feature from the data handler must be topography '
+               'to train the WindCC model, but received output features: {}'
+               .format(output_features))
+        assert output_features[-1] == 'topography', msg
+        output_features.remove('topography')
+        kwargs['output_features'] = output_features
+        super().set_model_params(**kwargs)
+
+    def _reshape_norm_topo(self, hi_res, hi_res_topo, norm_in=True):
```

Comment:
Similar comments here as on conditional_moments.py - We should move lots of this code into a super class since WindGan can inherit much of it as well. Maybe call the super class BaseWindInterface or something.